README.md

```diff
@@ -12,6 +12,8 @@ Nessie supports Iceberg Tables, Delta Lake Tables and Sql Views.
 
 More information can be found at [projectnessie.org](https://projectnessie.org/).
 
+Nessie is focused on working with the widest range of tools possible. [Check feature-matrix for further info on Tools & Integrations](https://projectnessie.org/tools/#feature-matrix)
```

Comment:
I would probably add that at L11 and change the line to: 
`Nessie supports Iceberg Tables, Delta Lake Tables and Sql Views. Additionally, Nessie is focused on working with the widest range of tools possible, which can be seen in the [feature matrix](https://projectnessie.org/tools/#feature-matrix).`